src/templates/aws/advanced.ts

```diff
@@ -21,7 +21,11 @@ export default class Advanced {
 
   private applyTemplate(): void {
     this.applyCommon()
-    this.applyVPC()
+    this.applyVpc()
```

Comment:
To avoid this file getting fatter, should we create the `addons` folder and put each code block into its folder corresponding?
For example: 
```
template/
├─ aws/
│  ├─ advanced.ts
│  ├─ addons/
│  │  ├─ vpc.ts
│  │  ├─ ssm.ts
│  │  ├─ s3.ts
```

Then in this file, we can import functions and call them 🤔

Reply:
I don't think I have time to work on that today. I think I can open another PR for that. We can merge this PR so Junan can use the fixes we made to the file helper on his PR. What do you think?

Reply:
I don't think I have time to work on that today. I think I can open another PR for that. We can merge this PR so Junan can use the fixes we made to the file helper on his PR. What do you think?

Reply:
I'm ok with that. But please create an issue for that to work later 🙏

Reply:
issue created here https://github.com/nimblehq/infrastructure-templates/issues/38